packages/cli/src/utils/base-command.js

```diff
@@ -59,30 +66,27 @@ class BaseCommand extends Command {
   get accessToken() {
     // define auth client
     const authClient = new AuthClient()
-
+    const { token } = this.flags
     return (new Promise((resolve, reject) => {
-      // validate existing access token
-      authClient.notify("validate_access_token", { accessToken: this._accessToken })
-        .then(res => {
-          if (!Object.keys(res || {}).length) {
-            // exchange refresh token for new access token
-            // in case of invalid access token
-            authClient.notify("exchange_refresh_token", { refreshToken: this.refreshToken })
-              .then(token => {
-                // re-write new access token and current refresh token
-                fs.writeFileSync(BaseCommand.CREDS_PATH, JSON.stringify({ access_token: token.access_token, refresh_token: token.refresh_token}))
-
-                // resolve promise
-                // with new refresh token and access token
-                resolve(token)
-              })
-              .catch(err => reject(err))
-            } else {
-              resolve({ id_token: res.access_token, access_token: res.access_token })
-            }
-
-        })
-        .catch(err => reject(err))
+      if(token) {
+        return this.exchangeToken(token, resolve, reject)
+      } else {
+          // validate existing access token
```

Comment:
too many spaces indentation is wrong

Reply:
I have checked the spaces here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else
i think this is the proper way